### DISCUSSION TITLE
MQTT payload for frigate/frigate_cam/person/snapshot wasn't text. Skipping...

### DISCUSSION THREAD

**Discussion by Minglarn** [blakeblackshear/frigate#2310] [Ask A Question] [answered]
I've got these in my loggs quite often...
What does this mean?
`MQTT payload for frigate/frigate_cam/person/snapshot wasn't text. Skipping...`

**Comment by blakeblackshear** [comment, marked as answer]
Do you have debug logging enabled? These are normal messages for the websocket to mqtt bridge.

**Reply by Minglarn** [reply]
Yes.. Debug is enabled at the moment.. 
Was a bit worried bout "snapshot wasn't text. Skipping..." .. Thought that it's an error on my side... 👍
